### DISCUSSION TITLE
v0.28.0-rc2

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#1625] [General]
* The cuda backend option multi_stream is now off by default. You should
  consider setting it to on if you have a recent gpu with a lot of vram.
* Updated default parameters.
* Newer and stronger nets are included in the release packages.
* Added support for onnx network files and runtime with the "onnx" backend.
* Several bug and stability fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.28.0-rc2'>v0.28.0-rc2</a>.</em>

**Comment by GosseRomkes** [comment]
Good update on my system! Thx a lot. Gosse

**Comment by giauvn** [comment]
Version lc0-v0.28.0-windows-cpu-openblas  cannot run after loading Gui Shark 2.2.

**Reply by borg323** [reply]
We need more information to be able to help.
